CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+# v3.1.0
+
+### Breaking
+
+- `disable_recipient_thread_close` is removed, a new configuration variable `recipient_thread_close` replaces it which defaults to False.
```

Comment:
There is no point in introducing this breaking change. Breaking changes require a major version bump

Reply:
This is to make the config var consistent with the others